engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -222,6 +226,10 @@ class WorkflowManagerActor(config: Config,
       log.error("Error writing to JobStore from WorkflowManagerActor: {}", t)
       // This is minorly bad. The JobStore would never rid itself of this workflow's entries. Unlikely to be a big deal
       stay()
+    case Event(EngineStatsCommand, data) =>
+      val sndr = sender()
+      context.actorOf(EngineStatsActor.props(data.workflows.values.toList, sndr), s"EngineStatsActor-${sndr.hashCode()}")
```

Comment:
Yeah it got kinda gnarly when I was playing around w/ it. Since I want to replace spray anyways I don't think that it's worth fighting our PerRequest structure